agent/agent.go

```diff
@@ -28,7 +28,7 @@ import (
 
 const (
 	defaultAgentHandshakeAttempts       = 5
-	defaultAgentHandshakeTimeoutMillis  = 500
+	defaultAgentHandshakeTimeoutMillis  = 5000
```

Comment:
How did you come to this increased timeout being more appropriate?

Reply:
Given that `500` milliseconds seemed sufficient, `5000` should be more than sufficient. I hard-coded a reduced number of CPUs available to the nex node to simulate the resources made available in GCP during testing.